README.md

```diff
@@ -374,6 +388,12 @@ Normally, setting required DSN information would be enough.
   }`
   - Specified keys will override common Sentry options for client sentry plugin
 
+### publicRuntimeConfigKey
+- Type: `String`
+  - Default: If Nuxt >=2.13 `sentry` , else `undefined`
+  - Specified object in Nuxt config in `publicRuntimeConfig[publicRuntimeConfigKey]` will override config at runtime. See documentation at https://fr.nuxtjs.org/docs/2.x/configuration-glossary/configuration-runtime-config/
```

Comment:
Please link using canonical URL https://nuxtjs.org/docs/2.x/configuration-glossary/configuration-runtime-config/
The current one is french.